plugins/scaffolder-backend/src/scaffolder/actions/builtin/helpers.ts

```diff
@@ -24,16 +24,26 @@ import { assertError } from '@backstage/errors';
 export type RunCommandOptions = {
   command: string;
   args: string[];
+  options: object;
```

Comment:
Hmm this is a breaking change unfortunately. You could add this as the last argument instead and make it optional to make it non-breaking. Also would be better to actually type this in someway and use the type from node to work out what is an option that you can pass through to the `spawn` command.

Reply:
> You could add this as the last argument instead

Last argument was already optional so I think this should be ok?

Added type information

Reply:
Ah sorry I misread that this was an object already, yep this is better now.